### PR TITLE
Improve upgrade with ">>" flows file

### PR DESCRIPTION
**Pull request by tituspijean**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -87,9 +87,14 @@ fi
 
 # Flows were stored in file named after the hostname.
 # Not very portable. Let's fix that.
-if [ ! -f "$final_path/data/flows.json" ]; then
+if [[ ! -f "$final_path/data/flows.json" && -f "$final_path/data/flows_$(hostname).json" ]]; then
 	mv "$final_path/data/flows_$(hostname)_cred.json" "$final_path/data/flows_cred.json"
 	mv "$final_path/data/flows_$(hostname).json" "$final_path/data/flows.json"
+# Flows could be stored in a file named '>>'.
+# Definitely weird, let's fix that.
```
Comment (fflorent) [inline scripts/upgrade:94]: Do you have any clue why it would have been stored in such a name?

Comment (tituspijean) [reply scripts/upgrade:94]: Yup... a very badly written systemd file, ending with `>> ...` in an attempt to redirect any output to the log file.  (That's the only location I had found those two characters)

**Comment by fflorent** [comment]
LGTM

**Comment by tituspijean** [comment]
!testme

**Pull request merged**